model/metrics/v8js-metrics.yaml

```
@@ -0,0 +1,96 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
+
+  - id: metric.veightjs.memory.total
+    type: metric
+    metric_name: v8js.memory.total
+    brief: "Total memory size pre-allocated."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+    attributes:
+      - ref: v8js.memory.type
+        requirement_level: recommended
+      - ref: v8js.heap.space.name
+        requirement_level:
+          conditionally_required: If memory type is `heap`.
+
+  - id: metric.veightjs.memory.used
+    type: metric
+    metric_name: v8js.memory.used
+    brief: "Used memory size."
+    instrument: updowncounter
+    unit: "By"
+    stability: experimental
+    attributes:
+      - ref: v8js.memory.type
+        requirement_level: recommended
+      - ref: v8js.heap.space.name
+        requirement_level:
+          conditionally_required: If memory type is `heap`.
+
+  - id: metric.veightjs.heap.space.size
+    type: metric
+    metric_name: v8js.heap.space.size
```

It seems to be a total size according to the brief, can we reflect it in the name?
E.g. 
```suggestion
    metric_name: v8js.heap.space.limit
```